src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/styles.tsx

```diff
@@ -47,9 +47,16 @@ const GridCell = styled('div')<{
     p.hasError &&
     `
       background: #fffcfb;
-      border-top: 1px solid #fa4747;
-      border-bottom: 1px solid #fa4747;
-      z-index: ${p.theme.zIndex.breadcrumbs.gridCellError};
+      border-bottom: 1px solid ${p.theme.red400};
```

Comment:
Do you need both the bottom border and the generated :after element?

Reply:
yes... it is a bit hacky..but... this one is for the bottom and the `:after` is for the border on the top  .. I used after instead of before because I am already using it in the `GridCellLeft`